features/org.eclipse.pde-feature/build.properties

```diff
@@ -11,6 +11,6 @@
 # Contributors:
 #     IBM Corporation - initial API and implementation
 ###############################################################################
-bin.includes=\
-feature.xml,\
-feature.properties
+bin.includes = feature.xml,\
```

Comment:
Not sure if the build failure is related, but this file should probably be left unchanged.